ondewo/vtsi/voip.proto

```diff
@@ -274,6 +282,19 @@ message StartCallInstanceResponse {
   bool success = 2;
 }
 
+// collection of calls to start
+message StartMultipleCallInstancesRequest {
+  repeated StartCallInstanceRequest requests = 1;
+}
+
+// response from the voip manager concerning the multiple started calls
+message StartMultipleCallInstancesResponse {
+  // request sent to the voip manager
+  StartMultipleCallInstancesRequest request = 1;
+    // success status
+  bool success = 2;
```

Comment:
so this general success is essentially a product of all individual successes, just for convenience to not iterate over them, correct?

Reply:
yes, if one fails, the result is failed and the further calls won't be executed

Reply:
Let's add this to the comment.